code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,330 @@
+		SEND_SIGNAL(src, COMSIG_CURE_MENTAL, src)
+
+
+/datum/brain_trauma/psychological/depression
+	name = "Depression"
+	desc = "Extensive damage to receptors in brain result in subject's neigh constant bad mood"
+	scan_desc = "Electric imaging suggests a mental disorder is present"
+	gain_text = "<span class='warning'>You feel emptier inside!</span>"
+	lose_text = "<span class='notice'>You feel whole once again.</span>"
+	cure_points = 1500
+	lithiated_mod = -2
+	control_mod = -1
+	fearless_mod = 4
+	relaxed_mod = 3
+
+/datum/brain_trauma/psychological/depression/on_gain()
+	ADD_TRAIT(owner, TRAIT_DEPRESSION, PSYCH_TRAIT)
+	..()
+
+/datum/brain_trauma/psychological/depression/on_lose()
+	REMOVE_TRAIT(owner, TRAIT_DEPRESSION, PSYCH_TRAIT)
+	..()
+
+/datum/brain_trauma/psychological/depression/on_life()
+	..()
+	if(HAS_TRAIT(owner, TRAIT_FEARLESS))
+		return
+	if(prob(5)) //this is much much worse then before!
+		SEND_SIGNAL(owner, COMSIG_ADD_MOOD_EVENT, "depression", /datum/mood_event/depression)
+
+
+/datum/brain_trauma/psychological/social_anxiety
+	name = "Social Anxiety"
+	desc = "Talking to people is very difficult for you, and you often stutter or even lock up."
+	gain_text = "<span class='danger'>You start worrying about what you're saying.</span>"
+	lose_text = "<span class='notice'>You feel easier about talking again.</span>" //if only it were that easy!
+	scan_desc = "Electric imaging suggests a mental disorder is present"
+	var/dumb_thing = TRUE
+	cure_points = 500
+	lithiated_mod = -2
+	control_mod = 2
+	fearless_mod = 1
+	relaxed_mod = 4
+
+/datum/brain_trauma/psychological/social_anxiety/on_life()
+	..()
+	if(HAS_TRAIT(owner, TRAIT_FEARLESS))
+		return
+	var/nearby_people = 0
+	for(var/mob/living/carbon/human/H in oview(3, owner))
+		if(H.client)
+			nearby_people++
+	var/mob/living/carbon/human/H = owner
+	if(prob(2 + nearby_people))
+		H.stuttering = max(3, H.stuttering)
+	else if(prob(min(3, nearby_people)) && !H.silent)
+		to_chat(H, "<span class='danger'>You retreat into yourself. You <i>really</i> don't feel up to talking.</span>")
+		H.silent = max(10, H.silent)
+	else if(prob(0.5) && dumb_thing)
+		to_chat(H, "<span class='userdanger'>You think of a dumb thing you said a long time ago and scream internally.</span>")
+		dumb_thing = FALSE //only once per life
+		if(prob(1))
+			new/obj/item/reagent_containers/food/snacks/spaghetti/pastatomato(get_turf(H)) //now that's what I call spaghetti code
+
+
+/datum/brain_trauma/psychological/social_anxiety/on_gain()
+	ADD_TRAIT(owner, TRAIT_SOCIAL_ANXIETY, PSYCH_TRAIT)
+	..()
+
+/datum/brain_trauma/psychological/social_anxiety/on_lose()
+	REMOVE_TRAIT(owner, TRAIT_SOCIAL_ANXIETY, PSYCH_TRAIT)
+	..()
+
+/datum/brain_trauma/psychological/bipolar
+	name = "Bipolar Disorder"
+	desc = "Subject experiences alternating mood swings"
+	gain_text = "<span class='danger'>You feel as on a moodswing.</span>"
+	lose_text = "<span class='notice'>You feel peace</span>" //if only it were that easy!
```

Comment:
period

also, you might wanna say something involving the words "balance" or "balanced" here